tasks/map/control/joystick.py

```diff
@@ -13,6 +14,36 @@
 from tasks.map.assets.assets_map_control import *
 
 
+def build_technique_points_strict(button: ButtonWrapper):
+    kwargs = {}
+    for trial, b in button.data_buttons.items():
+        x, y, x_, y_ = b.area
+        qw = int((x_ - x) / 4)
+        qh = int((y_ - y) / 4)
+        area = x + qw, y + qh, x_ - qw, y_ - qh
+        kwargs[trial] = Button(b.file, area, b.search, b.color, b.button)
+    return ButtonWrapper(f'{button.name}_STRICT', **kwargs)
+
+
+def build_technique_points_loose(button: ButtonWrapper):
+    kwargs = {}
+    for trial, b in button.data_buttons.items():
+        x, y, x_, y_ = b.area
+        search = x - 50, y - 5, x_ + 50, y_ + 5
+        kwargs[trial] = Button(b.file, b.area, search, b.color, b.button)
+    return ButtonWrapper(f'{button.name}_STRICT', **kwargs)
+
+
+TECHNIQUE_POINT_LOOSE_0 = build_technique_points_loose(TECHNIQUE_POINT_0)
+TECHNIQUE_POINT_LOOSE_1 = build_technique_points_loose(TECHNIQUE_POINT_1)
+
+TECHNIQUE_POINT_STRICT_1 = build_technique_points_strict(TECHNIQUE_POINT_1)
+TECHNIQUE_POINT_STRICT_2 = build_technique_points_strict(TECHNIQUE_POINT_2)
+TECHNIQUE_POINT_STRICT_3 = build_technique_points_strict(TECHNIQUE_POINT_3)
+TECHNIQUE_POINT_STRICT_4 = build_technique_points_strict(TECHNIQUE_POINT_4)
+TECHNIQUE_POINT_STRICT_5 = build_technique_points_strict(TECHNIQUE_POINT_5)
```

Comment:
Add `TECHINQUE_POINT_1.SEARCH.png` to enlarge search area, dont dynamically build buttons